go.mod

```diff
@@ -24,6 +24,7 @@ require (
 )
 
 require (
+	github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
Please also add this to the notice file

Reply:
This dependency is still not included in the suite-connector's binary because we use the legacy library